WcaOnRails/config/locales/en.yml

```
@@ -1363,6 +1377,9 @@ en:
       claim_wca_id_html: "If this is not your first competition, associate your WCA ID to your WCA account %{here}."
       register_link_html: "Register for this competition %{here}."
       register_below_html: "Register for this competition below."
+      part_of_a_series: "This competition is part of a Series Competition."
+      part_of_a_series_list: "This competition is part of the %{name} Series Competition. Other events in this series are:"
```

Yay! This is going to be a great help for Delegates!!!

small Nit :) 
```suggestion
      part_of_a_series_list: "This competition is part of the %{name} Series Competition. Other Competitions in this series are:"
```